pom.xml

```diff
@@ -73,12 +73,12 @@
     <dependency>
       <groupId>org.folio</groupId>
       <artifactId>folio-di-support</artifactId>
-      <version>1.1.0</version>
+      <version>1.6.0</version>
     </dependency>
```

Comment:
There are more folio dependencies that should be upgraded:
https://github.com/folio-org/folio-kafka-wrapper/releases/tag/v2.6.0
https://github.com/folio-org/data-import-processing-core/releases/tag/v3.3.3
https://github.com/folio-org/mod-data-import-converter-storage/releases/tag/v1.13.3